index.css

```diff
@@ -364,12 +387,59 @@ html {
   object-fit: cover;
   width: 100%;
   height: 100%;
+  display: block;
+}
+
+@media (min-width: 768px) {
+  .perks-row {
+    display: flex;
+    flex-flow: row nowrap;
+  }
+
+  .perks-row:nth-child(2n + 1) {
+    flex-direction: row-reverse;
+  }
+
+  .perks-col {
+    flex: 1 1 50%;
+  }
+
+  .perks-col.text {
+    padding: 25px;
+  }
+}
+
+/*
+ * ENQUIRE
+ */
+
+.bg-wrapper#enquire {
+  background: #2c383e;
+  text-align: left;
+}
+
+.bg-wrapper#enquire h1,
```

Comment:
Please don't use id in css.
give the wrapper a specific class like `enquire-wrapper` and style the class